src/App.tsx

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
 
 import './App.scss';
+import PostList from './components/PostList/index';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+const preparedPosts: Post[] = posts.map(post => {
+  const user = users.find(person => person.id === post.userId) || null;
+
+  const comment = comments.filter(message => message.postId === post.id) as Comment[];
```

Comment:
It'd be easier to name elements of arrays that you iterating as a singular version of array name. E.g. `users.find(user` not `person`, same for comments